src/index.php

```diff
@@ -122,11 +122,14 @@
 // Init Sorting variables
 $content['sorting'] = "";
 $content['searchstr'] = "";
-$content['searchstr_htmlform'] = "";
+$content['searchstr_htmlform'] = "datelastx:6 severity:-8";
 $content['highlightstr'] = "";
 $content['highlightstr_htmlform'] = "";
 $content['EXPAND_HIGHLIGHT'] = "false";
 
+// hide header by default
+$_SESSION['SESSION_MAXIMIZED'] = true;
+
```

Comment:
Same as with AUTORELOAD_ID, have a configuration variable defined in the config.sample.php template, and write this default into $_SESSION['SESSION_MAXIMIZED']. Also you need to define a default for SESSION_MAXIMIZED if the configuration variables is not defined.